src/test/java/org/folio/rest/impl/integrationsuite/EholdingsAssignedUsersImplTest.java

```diff
@@ -152,56 +156,40 @@ public void shouldReturn422OnPostWhenAssignedUserDoesNotHaveRequiredParameters()
 
     AssignedUserPostRequest assignedUserPostRequest = new AssignedUserPostRequest().withData(expected);
     String postBody = Json.encode(assignedUserPostRequest);
-    Errors errors = postWithStatus(resourcePath(credentialsId), postBody, SC_UNPROCESSABLE_ENTITY).as(Errors.class);
+    String endpoint = String.format(ASSIGN_USER_PATH, credentialsId);
+    Errors errors = postWithStatus(endpoint, postBody, SC_UNPROCESSABLE_ENTITY).as(Errors.class);
 
     assertThat(errors.getErrors(), hasSize(5));
     assertThat(errors.getErrors(), everyItem(hasProperty("message", equalTo("may not be null"))));
   }
 
   @Test
-  public void shouldReturn204OnPutWhenAssignedUserIsValid() {
+  public void shouldReturn204OnDeleteUserAssignment(){
```

Comment:
This tests for `put` are needed.

Reply:
probably during the rebase I accidentally removed them. Thank you for note, added them back